src/com/facebook/buck/android/toolchain/ndk/impl/NdkCxxPlatforms.java

```diff
@@ -984,11 +999,29 @@ Path getNdkToolRoot() {
       }
     }
 
+    boolean isUnifiedHeaders() {
+      return unifiedHeaders || ndkMajorVersion >= 16;
```

Comment:
I'd rather throw an error rather than silently disable feature especially given that non-unified headers may be wrong and unified headers are generated from the Android platform.

Also, unified headers are supported from v14.